appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
```

Comment:
I've just left it from your's appveyor branch ;)
I guess we could check several versions